app.ts

```diff
@@ -0,0 +1,319 @@
+let fromNumber = 0;
+let recordNumberTotal: number;
+let count = 0;
+let timeout = 0;
+
+window.onload = () => {
+	createNavigation();
+	recordCount();
+	headingRowCreation();
+	fromNumber = 0;
+	new pageNavigation();
```

Comment:
You should create the functions and classes before you use them (They should appear before the `window.onload` line)